### PR TITLE
ember: add page

### PR DESCRIPTION
**Pull request by alexlafroscia**

### CODE REVIEW

**Comment by agnivade** [comment]
I actually prefer the current description. It is to the point and describes accurately what the tool does. In general, we tend to avoid using subjective words like "ambitious". And mentioning "command line interface for web applications" does not provide much useful information.

File: pages/common/ember.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Run the development server:
+
+`ember serve`
+
+- Run the test suite:
+
+`ember test`
+
+- Generate files:
```
Comment (agnivade) [inline pages/common/ember.md]: Can you expand this a bit further ? It's not exactly clear what you mean by generate files.

**Comment by alexlafroscia** [comment]
@jimfilippou yeah, that might be a bit more clear. I grabbed the tagline from [the Github repo](https://github.com/ember-cli/ember-cli) and used that.

File: pages/common/ember.md (hunk 1 above)
Comment (alexlafroscia) [reply pages/common/ember.md]: For sure. I actually just grabbed the headings and examples from the Ember CLI readme, but I can definitely expand it more.

**Comment by sbrl** [comment]
Looks ok to me! Thanks, @alexlafroscia 😺

**Pull request merged**